index.html

```diff
@@ -1,80 +1,87 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rock, Paper, Scissors</title>
-</head>
+  </head>
 
-<body>
+  <body>
 
-<script>
-  
-function computerPlay() {
-    let rand = Math.random();
-    if (rand < 0.34) {
-        return 'rock';
-    }else if(rand <= 0.67) {
-        return 'paper';
-    }else{
-        return 'scissors';
-    }
-}
+    <script>
+      function computerPlay() {
+        const rand = Math.random();
+        if (rand < 0.34) {
+          return 'rock';
+        } else if (rand <= 0.67) {
+          return 'paper';
+        } else {
+          return 'scissors';
+        }
+      }
 
-let playerScore = parseInt(0);
-let computerScore = parseInt(0);
+      let playerScore = parseInt(0);
+      let computerScore = parseInt(0);
 
-function playRound(playerSelection, computerSelection) {
-if(playerSelection.toLowerCase() === 'rock'){
-    if(computerSelection === 'paper'){
-        computerScore++;       
-    }else if(computerSelection === 'scissors'){
-        playerScore++;       
-    }else{
-        return;
-    }
-}
+      function playRound(playerSelection, computerSelection) {
 
-if(playerSelection.toLowerCase() === 'paper'){
-    if(computerSelection === 'paper'){
-        return;
-    }else if(computerSelection === 'scissors'){
-        computerScore++;
-    }else{
-        playerScore++;
-    }
-}
+        if (playerSelection === 'rock') {
+          switch (computerSelection) {
```

Comment:
I like what you did here, but you should still be able to reduce the nesting so that it's just the one set of conditional statements without any nesting conditionals or switch statements